src/main/webapp/WEB-INF/web.xml

```diff
@@ -1,5 +1,8 @@
 <web-app xmlns="http://xmlns.jcp.org/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd" version="3.1">
     <display-name>COVID Pulse Application</display-name>
+    <welcome-file-list>
```

Comment:
Same as my home servlet comment, can we make the index.jsp our welcome file?

Reply:
I'll work on it.